Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -1676,6 +1675,10 @@ - (void)_updateLongPressAndHighlightLongPressGestures
     // We only enable the long press gesture in the case where the app is linked on iOS 12 or earlier (and
     // therefore prefers the legacy action sheet over context menus), and link previews are also enabled.
     [_longPressGestureRecognizer setEnabled:!self._shouldUseContextMenus && self.webView.allowsLinkPreview];
+#else
+    [_highlightLongPressGestureRecognizer setEnabled:NO];
+    [_longPressGestureRecognizer setEnabled:NO];
```

Comment:
@whsieh Can you confirm this is okay to do for watchOS?

Reply:
Yep, should be safe — we don't use context menus/LPGR for anything there.